src/index.html

```diff
@@ -11,6 +11,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    method="get"
+    action="/"
+    data-qa="big">
```

Comment:
```suggestion
    <form
      method="get"
      action="/"
      data-qa="big"
    >
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/225234954-cd7f5186-8731-4731-82df-7cca5df751b6.png)